cognite_toolkit/_cdf_tk/commands/dump_data.py

```diff
@@ -244,6 +253,55 @@ def _get_resource_columns(self) -> list[SchemaColumn]:
         return columns
 
 
+class FileMetadataFinder(AssetCentricFinder[FileMetadata]):
+    supported_formats = frozenset({"csv", "parquet"})
+
+    def _create_loader(self, client: ToolkitClient) -> ResourceLoader:
+        return FileMetadataLoader.create_loader(client)
+
+    def _aggregate_count(self, hierarchies: list[str], data_sets: list[str]) -> int:
+        result = self.client.files.aggregate(
+            filter=FileMetadataFilter(
+                data_set_ids=[{"externalId": item} for item in data_sets] or None,
+                asset_subtree_ids=[{"externalId": item} for item in hierarchies] or None,
+            )
+        )
+        return result[0].count if result else 0
```

Comment:
What's the type for "result" here? Is it FileMetadataList from the SDK?
Wondering what will happen if the indexed result has no "count" field.